test/contexts/__snapshots__/SignedInUserContext.test.jsx.snap

```diff
@@ -2,8 +2,6 @@
 
 exports[`SignedInUserContext > fetches user data if user is logged in 1`] = `
 <div>
-  <div>
-    https://example.com/pod/
-  </div>
+  <div />
```

Comment:
What's this update?

Reply:
I believe this has to do with the new updateActivity function for the profile card inside useEffect. When including it, says the container does not match the snapshot.

<img width="1440" alt="Screen Shot 2023-06-26 at 12 08 00" src="https://github.com/codeforpdx/PASS/assets/14917816/95cfb6b2-b392-4751-bbef-b14831a42ac8">

So I've updated the snapshot with `u`.

<img width="1440" alt="Screen Shot 2023-06-26 at 12 10 06" src="https://github.com/codeforpdx/PASS/assets/14917816/f5a60ae4-28a3-499f-a13c-b07c89d70488">

Without it, it's the same as before asking for `https://example.com/pod/` in that location.

<img width="1440" alt="Screen Shot 2023-06-26 at 12 00 32" src="https://github.com/codeforpdx/PASS/assets/14917816/0310775e-2672-4ba3-91a6-3b937aa78e58">

Should I revert the snapshot back?

Reply:
For now, I've reverted the snapshot back to the original form in Development.

Reply:
If it's affecting the rendering of the snapshot, and you're not sure why, there's probably a bug.

Although I think there may be a pre-existing issue with that test. It seems like it's not reading the mocked ENV properly. I'd leave this snapshot as-is for now, and we can fix that issue in another PR.

Reply:
Got it, I'll leave it alone. I'll start an issue for it stating it could be a bug.